go/libkb/errors.go

```diff
@@ -1399,6 +1399,12 @@ func (e UnknownStreamError) Error() string {
 	return "unknown stream format"
 }
 
+type UnicodeUnsupportedError struct{}
+
+func (e UnicodeUnsupportedError) Error() string {
+	return "unicode not supported"
```

Comment:
UTF-16 is used in Java too according to http://unicode.org/faq/utf_bom.html
Also, I hadn't realized the Unicode spec allows for a UTF-8 BOM, which I assume we would probably choke on as well? I wonder if that will ever come up:
https://en.wikipedia.org/wiki/Byte_order_mark#UTF-8

I added UTF-16 conversion for uploading some of the installer logs, it isn't so hard. Maybe we could have an explicit flag to accept a UTF-16 saltpack.